.circleci/config.yml

```diff
@@ -51,7 +51,7 @@ jobs:
               # Should only happen when re-running a job, and the workspace is gone
               time make build test-bins
             fi
-            make docker.all generate_yaml
+            make SKIP_REBUILD_CHECK=1 docker.all generate_yaml
```

Comment:
can you add a comment above this line, on what this variable is doing? so that someone else doesn't accidentally delete it.

Reply:
The other approach I considered was to stick a "SKIP_REBUILD_CHECK=1" line in a .istiorc.mk file (the top-level Makefile already tries to opportunistically include this file).  This would keep things cleaner in this file and avoid cases where people forget to add this to new rules (or aren't sure if/when/where to include it [it's basically fine to use it everywhere but I just tried to add it to places where it looked like the script was trying to do a 2nd/subsequent rebuild]).

However, to be the most turnkey I'd probably have to have a line like "-include $(HOME)/.istiorc-global.mk" (and have the file in the docker image), or try to set up an environment variable that applies globally to the yml file (if there's a way to express that).